diff --git a/dist/graphql/graphql.js b/dist/graphql/graphql.js
index 84c6eabf5fe4a21e577bfb883582ec35b0677fa5..2391d0c2e916c48a5340e79bc17efa58f7fc5b65 100644
--- a/dist/graphql/graphql.js
+++ b/dist/graphql/graphql.js
@@ -45,6 +45,11 @@ async function executeQuery(gqlHandler, query, variables) {
  */
 async function getGqlHandler() {
   const gqlPath = _path.default.join((0, _projectConfig.getPaths)().api.functions, 'graphql');
+
+  console.log('-- DATABASE_URL --')
+  console.log('DATABASE_URL', process.env.DATABASE_URL)
+  console.log('-- DATABASE_URL --')
+
   try {
     const {
       handler
